.github/workflows/pytorch-linux-xenial-py3.6-gcc5.4.yml

```diff
@@ -0,0 +1,147 @@
+# WARNING THIS IS AUTOGENERATED DO NOT UPDATE MANUALLY PLS
+# Template is at:    .github/templates/linux_ci_workflow.yml
+# Generation script: .github/scripts/generate_linux_ci_workflows.py
+name: Linux CI (pytorch-linux-xenial-py3.6-gcc5.4)
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - release/*
+  workflow_dispatch:
+
+env:
+  BUILD_ENVIRONMENT: pytorch-linux-xenial-py3.6-gcc5.4
+  DOCKER_IMAGE_BASE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-gcc5.4
+  SCCACHE_BUCKET: ossci-compiler-cache-circleci-v2
+  TORCH_CUDA_ARCH_LIST: 5.2
+  IN_CI: 1
+
+jobs:
+  calculate-docker-image:
+    runs-on: ubuntu-18.04
+    outputs:
+      docker_image: ${{ steps.calculate-tag.outputs.docker_image }}
+    steps:
+      - name: Checkout PyTorch
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
It's too bad you have to do the PyTorch checkout action twice (once here, and once below). Wonder if there's a GH API/GraphQL call that can do this without a checkout :o)

Reply:
Unfortunately since they do run on separate runners the checkout step is still necessary :/